src/controller.rs

```diff
@@ -16,6 +16,8 @@ use gzip::Gzip;
 
 const CRASH_PING_RETRIES: u32 = 10;
 const CRASH_PING_WAIT: u32 = 500;
+const CRASH_PING_TYPE: &'static str = "/cd-crash/";
+const TELEMETRY_SERVER_URL: &'static str = "https://incoming.telemetry.mozilla.org/submit/telemetry/";
```

Comment:
I guess the 
     &'static
is necessary since you are declaring a static string literal. At least that's what I've just seen when I looked it up. Strange syntax but that's rust, I guess.